docs/src/get-started/goals.rst

```diff
@@ -0,0 +1,98 @@
+.. _userdoc-goals:
+
+Goals of this library
+=====================
+
+With the creation of equistore, we have three main use cases in mind:
+
+1. provide an interchange format for the atomistic machine learning ecosystem,
+   making different players in this ecosystem more interoperable with one
+   another;
+2. make it easier and faster to prototype new machine learning representations,
+   models and algorithms applied to atomistic modeling;
+3. run large scale simulations using machine learning interatomic potentials,
+   with fully customizable potentials, directly defined by the researchers
+   running the simulations.
+
+
+Equistore as an interchange format
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+First, equistore is an interchange format for different libraries in the
+atomistic machine learning ecosystem. There is currently an explosion of
+libraries and tools for atomistic machine learning, implementing new
+representation, new models, and advanced research technics. Unfortunately each
+one of these libraries lives mostly separated from the others, resulting in a
+lot of duplicated effort. With equistore, we want to provide a way for these
+libraries to communicate with one another, by giving everyone a *lingua franca*,
+a way to share data and metadata.
+
+This goal is enabled by multiple features of equistore: first, equistore allows
+storing data coming from many different sources, without requiring to first
+convert the data to a specific format. Currently, we support data stored inside
+numpy arrays, torch tensor (including tensors on GPU or other accelerators), as
+well as arbitrary user-defined C, C++, and Rust array types. A second part of
+this goal is achieved by also storing metadata together with the data,
+communicating between libraries exactly **what** is stored in the different
+arrays. We also store both data and gradients of this data with respect to
+arbitrary parameters together, enabling for example training of models using
+energy, forces and virial. Finally, we also make sure that the data storage is
+as efficient as possible and can exploit the inherent sparsity of atomistic
+data, in particular in gradients.
+
+As a developer a library in the atomistic machine learning ecosystem, you can
+provide conversion functions to and from equistore
+:py:class:`equistore.TensorMap` to enable using your library in conjunction with
+the rest of the equistore-compatible libraries!
```

Comment:
Where would these conversion functions live? That is unclear.

Reply:
I tried to clarify this